libs/types/src/permissions.rs

```diff
@@ -289,6 +295,7 @@ where
 				}
 				PoolRole::LoanAdmin => Ok(self.pool_admin.insert(PoolAdminRoles::RISK_ADMIN)),
```

Comment:
This is unrelated to this PR, but the naming seems a bit off here, not all pool roles are pool admin roles. I wonder if we should change `self.pool_admin` to `self.pool` and `PoolAdminRoles` to just `PoolRoles`. What do you think?

cc @mustermeiszer

Reply:
Sounds sane to me. I would be more pro having an additional struct like `PoolAdminRoles` and `PoolRoles` but this would mean we need to write a migration. So lets keep it simple.